src/ray/protobuf/gcs_service.proto

```diff
@@ -596,6 +608,8 @@ service NodeResourceInfoGcsService {
   rpc ReportResourceUsage(ReportResourceUsageRequest) returns (ReportResourceUsageReply);
   // Get resource usage of all nodes from GCS Service.
   rpc GetAllResourceUsage(GetAllResourceUsageRequest) returns (GetAllResourceUsageReply);
+  // Get current stats (e.g., pending tasks) of gcs server.
+  rpc GetGcsStats(GetGcsStatsRequest) returns (GetGcsStatsReply);
```

Comment:
we probably need to be more specific? Gcs has a lot of state and this only queries scheduler state.

Reply:
Sure, will do.